src/main/java/frc/robot/utilities/TOFSensorSimple.java

```diff
@@ -7,23 +7,45 @@
 import com.playingwithfusion.TimeOfFlight;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
 
 /** Add your docs here. */
 public class TOFSensorSimple {
   private TimeOfFlight sensor;
   private double threshold;
+  private boolean isBeamBrokeSim;
 
   public TOFSensorSimple(int ID, double threshold) {
     sensor = new TimeOfFlight(ID);
     this.threshold = threshold; // in mm
+    isBeamBrokeSim = false;
+  }
+
+  public TOFSensorSimple(int ID) {
+    sensor = new TimeOfFlight(ID);
+    this.threshold = 1; // in mm
+  }
+
+  public void updateSimBool(boolean isBeamBroke) {
+    isBeamBrokeSim = isBeamBroke;
   }
 
   public double getMilliMeters() {
     return sensor.getRange();
   }
 
   public boolean isBeamBroke() {
-    return getMilliMeters() < threshold;
+    if (Robot.isReal()) {
+      return getMilliMeters() < threshold;
+    }
+    return isBeamBrokeSim;
+  }
+
+  public boolean isBeamBrokeInverse() {
```

Comment:
Which would make this `isBeamUnbroken`.

Reply:
Also it looks like this method is never used; instead we're using `!isBeamBroke()`, so let's get rid of this one.